File: JAVA PROGRAMMING/Test28.java
// 28. PRINT SUM OF ALL ODD NUMBER BETWEEN RANGE1 TO RANGE2
import java.util.Scanner;
public class Test28 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range1 Value: ");
        int range1 = in.nextInt();
        System.out.println("Enter Range2 Value: ");
        int range2 = in.nextInt();
        int sum = 0;
        for(int i = range1; i <= range2; i++) {
            if(i % 2 != 0) {
                sum += i;
            }
        }
        System.out.println("Sum Of All Odd Numbers From "+range1+" To "+range2+" Is: "+sum);
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test28.ts
// 28. PRINT SUM OF ALL ODD NUMBER BETWEEN RANGE1 TO RANGE2
const func28 = (range1: number, range2: number): void => {
    let sum: number = 0;
    for(let i = range1; i <= range2; i++) {
        if(i % 2 !== 0) {
            sum += i;
        }
    }
    console.log(`Sum: ${sum}`);
}
func28(100, 200);
func28(50, 100);
func28(1, 50);